src/components/Content/contactUs/ContactForm.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { Container, Grid, FormControlLabel, Checkbox } from '@mui/material';
+import { NormalButton } from '@/components/Button/NormalButton';
+import { ControlledTextField } from '@/components/TextField/TextField';
+import { useForm, FormProvider } from 'react-hook-form';
+import { z } from 'zod';
+import { zodResolver } from '@hookform/resolvers/zod';
+import ControlledTypography from '@/components/Typography/Typography';
+import { requiredString } from "@/utils/formSchema";
+
+const schema = z.object({
+  firstName: requiredString("Your firstname is required."),
+  lastName: requiredString("Your lastname is required."),
+  email: requiredString("Your email is required.").email(),
+  message: requiredString("Message is required"),
+  pos: z.boolean(),
+  tms: z.boolean(),
+});
+
+const ContactForm = () => {
```

Comment:
Your code is too straight forward. Isolate your grid and textfield to different function example.

Also you should consider using 

- useFormContext inside your ContactUsForm

const ContactUsForm = () => {
//...here you can put the Grid and ControlledTextField
}


export const ContactUs = () => {
const form = useForm({})
return (
<FormProvider {...form}></FormProvider>
)
}